### PR TITLE
Complete first page in 'About the person' section

### PR DESCRIPTION
**Pull request by edavey**
We follow the link from the task list ("Complete equality and diversity monitoring") to the first page and indicate that "Yes" the person is willing to answer the equality and diversity questions.

### CODE REVIEW

File: steps/apply.ts [hunk 1]
```diff
@@ -35,3 +35,15 @@ export const completeFundingInformationTask = async (page: Page, name: string) =
   await fundingInformationPage.checkRadio('Personal money or savings')
   await fundingInformationPage.clickSave()
 }
+
+export const completeAboutThePersonTask = async (page: Page, name: string) => {
```
Comment (Harriethw) [inline steps/apply.ts:39]: query: should this be `completeEqualityAndDiversityTask`? The 'About the Person` is the section?

**Review by Harriethw — APPROVED** [review]
thanks Ed!

File: steps/apply.ts (hunk 1 above)
Comment (edavey) [reply steps/apply.ts]: good point! thank you

I've had another go at this and am now proposing "per-section" steps at the top level in our "Create application" feature. I've extracted the task-level step helpers to section-specific files. Hopefully this is pattern which will be clear to maintain and build on?

File: steps/apply.ts [hunk 2]
```diff
@@ -24,14 +26,10 @@ export const enterCrn = async (page: Page, crn: string) => {
...
+}
 
-  const fundingInformationPage = await ApplyPage.initialize(
-    page,
-    `How will ${name} pay for their accommodation and service charge?`,
-  )
-  await fundingInformationPage.checkRadio('Personal money or savings')
-  await fundingInformationPage.clickSave()
+export const completeAboutThePersonSection = async (page: Page, name: string) => {
+  await completeWillAnswerEqualityQuestionsTask(page, name)
```
Comment (patrickjfl) [inline steps/apply.ts:34]: sorry I don't have time for a full review, but I was just alerted by Harriet's comment via email. This should be completeEqualityAndDiversityMontitoringTask, shouldn't it? That's the task, and willAnswerEqualityQuestions is a page within that task, I think

**Pull request merged**